src/components/dashboard/elements/ButtonElement.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { FunctionComponent } from 'react';
+import globalize from '../../../scripts/globalize';
+
+const createButtonElement = ({ type, id, className, title, leftIcon, rightIcon }: IProps) => ({
+    __html: `<button
+        is="emby-button"
+        type="${type}"
+        ${id}
+        class="${className}"
+        >
+        ${leftIcon}
+        <span>${title}</span>
+        ${rightIcon}
+    </button>`
+});
+
+type IProps = {
+    type?: string;
+    id?: string;
+    className?: string;
+    title?: string;
+    leftIcon?: string;
+    rightIcon?: string;
+}
+
+const ButtonElement: FunctionComponent<IProps> = ({ type, id, className, title, leftIcon, rightIcon }: IProps) => {
```

Comment:
These seem to be pretty generic wrappers for the emby webcomponents now... Do you think it would make sense for them to live in `src/elements` alongside them?

Reply:
[c9fec5d](https://github.com/jellyfin/jellyfin-web/pull/3733/commits/c9fec5d7fd8d38dcbac50fc3b814583a81a27882)